io/plugins/eu.esdihumboldt.hale.io.gml.test/src/eu/esdihumboldt/hale/io/gml/reader/internal/StreamGmlReaderTest.groovy

```diff
@@ -163,4 +198,49 @@ class StreamGmlReaderTest {
 
 		gmlReader.getInstances();
 	}
+
+	@Test
+	public void testWfsFilter() {
+		def schemaLocation = getClass().getResource("/data/filter_doubles/filter.xsd").toURI();
+		def gmlLocation = getClass().getResource("/data/filter_doubles/filter.xml").toURI();
+		def paging = 1
+		def expected = 4
+
+		def sourceSchema = loadSchema(schemaLocation)
+		def instances = loadGmlFilter(gmlLocation, sourceSchema, null)
+
+		// Instantiate the class to test
+		DupplicateIDsFilterIterater dupplicateIDsFilterIterater = new DupplicateIDsFilterIterater(instances.iterator());
```

Comment:
@stempler could you please help me understand why I cannot import the package for DupplicateIDsFilterIterater? I have added the plugin in the MANIFEST.mf but somehow I cannot import the class.